Update ru_ru.json

Heimdallr-1 opened the pull request.

Commit:
Update ru_ru.json

TheIllusiveC4 commented on src/main/resources/assets/curios/lang/ru_ru.json:
```diff
@@ -1,21 +1,21 @@
+  "commands.curios.replace.success": "%s слот для %s заменён на %s",
+  "argument.entity.options.curios.description": "Инвентарь существа curios",
+  "argument.curios.type.unknown": "Неизвестный тип слота цурио %s",
+  "curios.identifier.curio": "Цурио",
```
I'll need some clarity on this change. My understanding is that this translates literally to "curio" whereas the previous translation is "accessory". The correct translation is "accessory" in terms of its meaning and not the name of the mod. It is only this way in English because curio has a similar meaning, but I don't know if this is the same for Russian. If it's not, then this should not be changed.

TheIllusiveC4 commented on src/main/resources/assets/curios/lang/ru_ru.json:
```diff
@@ -25,16 +25,16 @@
-  "curios.modifiers.charm": "Если надето как амулет:",
-  "curios.modifiers.hands": "При ношении на руках:",
-  "curios.modifiers.bracelet": "Если надето как браслет:",
+  "curios.modifiers.curio": "Когда надето в качестве цурио:",
```
Same comment as above in reference to "accessory" vs "curio".

Commit:
Update ru_ru.json

I looked up the meaning of the word Curio in the context of jewelry and accessories and I restored its former name.

TheIllusiveC4 commented:
Looks good, thanks!
